pkg/env/msiauthorizer.go

```diff
@@ -18,28 +20,37 @@ const (
 	MSIContextGateway MSIContext = "GATEWAY"
```

Comment:
I think we can delete this type, and the `msiContext` parameter on `NewMSIAuthorizer`.  If there's only one option then it's pointless.

Reply:
I think this is OK to do in a follow-up PR. We can throw those in with the RHCOS changes.